src/Microsoft.PowerShell.Commands.Management/commands/management/GetComputerInfoCommand.cs

```diff
@@ -558,13 +558,13 @@ private static MiscInfoGroup GetOtherInfo(CimSession session)
             // get secure-boot info
             //TODO: Local machine only? Check for that?
             FirmwareType fwType = FirmwareType.Unknown;
-            if (Native.GetFirmwareType(ref fwType))
+            if (GetFirmwareType(ref fwType))
```

Comment:
I don't see any need of returning a Boolean from the wrapper methods.  They already return a default value on error and we can get rid of the two assignment statements here.